.github/workflows/terraform.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Lokomotive Unit Tests'
```

Comment:
@alban just making sure you're aware we use Concourse as the CI platform for the project.

Reply:
I thought we could use GitHub actions for compilation check and unit tests that does not require to create any external nodes (just the ubuntu VM used by GitHub action). And we could keep Concourse for all other tests that actually install Lokomotive with external nodes with additional cloud credentials.

Reply:
I'm willing to discuss this, however I'd rather discuss the implications with the team first and not introduce a new element "implicitly" :-)